### PR TITLE
Added script to use DNS challenge for http.net DNS API

### PR DESCRIPTION
**Pull request by rkaldung**

### CODE REVIEW

**Comment by Neilpang** [comment]
please check this guide and fix the errors:
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by Neilpang** [comment]
please merge the latest code.

**Comment by rkaldung** [comment]
Merged the latest code as requested

File: dnsapi/dns_http_net.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```
Comment (Neilpang) [inline dnsapi/dns_http_net.sh]: it's not bash.

please check this guide : https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

File: dnsapi/dns_http_net.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+#_acme-challenge.www.domain.com
+#returns
+# _sub_domain=_acme-challenge.www
+# _domain=domain.com
+
+_get_root() {
+  domain=$1
+  i=2
+  while true; do
+    h=$(printf "%s" "$domain" | cut -d . -f $i-100)
```
Comment (Neilpang) [inline dnsapi/dns_http_net.sh]: please use "echo" instead of "printf" here

File: dnsapi/dns_http_net.sh [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        {
+            "name": "%s",
+            "type": "TXT",
+            "content": "\\"%s\\""
+        }
+    ]
+}' "$HTTP_NET_AUTHTOKEN" "$_domain" "$fulldomain" "$txtvalue")
+
+  response="$(_post "$payload" "${HTTP_NET_API}/zoneUpdate" "" "POST" "text/json")"
+  _debug response "$response"
```
Comment (Neilpang) [inline dnsapi/dns_http_net.sh]: use "_debug2"  here.

File: dnsapi/dns_http_net.sh [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            "name": "%s",
+            "type": "TXT",
+            "content": "%s",
+            "ttl": 300
+        }
+    ]
+}' "$HTTP_NET_AUTHTOKEN" "$_domain" "$fulldomain" "$txtvalue")
+
+  response="$(_post "$payload" "${HTTP_NET_API}/zoneUpdate" "" "POST" "text/json")"
+  _debug response "$response"
```
Comment (Neilpang) [inline dnsapi/dns_http_net.sh]: "_debug2"

**Comment by Neilpang** [comment]
please update readme:  https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by rkaldung** [comment]
I'll close this pull request in favor of the API hostingde, it's the same provider with a different endpoint.
@Neilpang Thank you for your efforts.

**Pull request closed without merging**